spec/Data_Parallelism.tex

```diff
@@ -257,15 +257,15 @@ \section{Forall Intents}
 affect the behavior of a task construct such as a \chpl{coforall} loop.
 \end{rationale}
 
-\begin{future}
-We would like to introduce "reduction" intents
-that will let us implement reductions using forall intents.
-\end{future}
-
-\begin{future}
-As with task intents, we may want to disallow some intents,
-for example \chpl{inout} and \chpl{out}.
-\end{future}
+\begin{craychapel}
+% "A limited set of" ?
+An initial implementation of "reduce" intents is also available,
+which will let us implement reductions using forall intents.
```

Comment:
Instead of rationalizing it for our purposes, how about changing this to a user's point of view.  E.g.,

"which permits users to reduce values across iterations of a forall loop"